### ISSUE TITLE
Accessibility fixes on the teacher view

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1296]
### HTML structure
(do not skip levels, and only go down a level to signify page structure)
- Subject listing page
- Learning theme should probably be a h2, or structurally moved to be within the unit listing section and then stay as a h3
- Teacher quizzes
    - "\<ul> and \<ol> must only contain \<li>, \<script> or \<template> elements."
- Lesson overview drop downs should be h2 to allow a user to understand the layout of a page. Currently the headings are Lesson title, core content and then the number of questions in a quiz. I fee like the number of questions in a quiz (currently h3) should not be a heading

### Roles
- Both learning theme and tier selection on ks4 maths 
    - We have navigation, tiers navigation and tiers navigation (I can show you the issue when using a screen reader)
- Checkboxes on search page
    - "IDs used in ARIA and labels must be unique"
    - I'm a bit confused about Axe reporting this one because it looks unique to me?
- Slide deck/worksheet on overview page
    - Axe complains that the role isn't a valid type, but it then suggest the current role (iframe) is what it wants...

### Other
- Can we remove the "quiz image" alt text from quiz images and replace it with an empty string so that the images are skipped by a screen reader. "quiz image" provides no extra context and is confusing
- We will want a "correct answer" label of some kind so that a screen reader user knows which answer is correct on the teacher view
